src/google_map.js

```diff
@@ -328,10 +328,12 @@ export default class GoogleMap extends Component {
     const mapDom = ReactDOM.findDOMNode(this.refs.google_map_dom);
     window.removeEventListener('resize', this._onWindowResize);
     window.removeEventListener('keydown', this._onKeyDownCapture);
-    mapDom.removeEventListener('mousedown', this._onMapMouseDownNative, true);
```

Comment:
Not sure. Seems like a mistake with inferno. I reported it here: https://github.com/trueadm/inferno/issues/590 . Not sure if/when it will be fixed.